roofit/roofit/src/RooBernstein.cxx

```diff
@@ -216,6 +218,24 @@ RooSpan<double> RooBernstein::evaluateBatch(std::size_t begin, std::size_t batch
   return output;
 }
 
+
+RooSpan<double> RooBernstein::evaluateSpan(BatchHelpers::RunContext& evalData, const RooArgSet* normSet) const {
+  RooSpan<const double> xData = _x->getValues(evalData, normSet);
+  const size_t batchSize = xData.size();
+  if (batchSize < 2) {
```

Comment:
@hageboeck so for a batch size of 1 (i.e. no batch) I don't get any results? I would have naively assumed that it would read `if (batchSize < 1)`?

Reply:
See also roofit/roofit/src/RooChebychev.cxx:243

Reply:
Not needed! A bug, actually!
The new API will always return a value. If there is no data for that object, it is supposed to return a batch of size 1 with the current value of the object. The leaves of the computation graphs are programmed to do exactly that:
https://github.com/root-project/root/blob/49d76a5b76e9fcdd783efe37d0d7c99d4c3df918/roofit/roofitcore/src/RooRealVar.cxx#L219-L237

Reply:
See also the documentation update I just put up:
https://github.com/root-project/root/pull/6803

Specifically:
https://github.com/root-project/root/pull/6803/commits/fccc4dc4f4000bcdb1e796cf82e34877745670c9